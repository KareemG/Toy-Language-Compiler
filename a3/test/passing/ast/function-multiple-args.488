% Declaring and calling a function that requires multiple arguments

{
    function test(a : integer, b : boolean) : integer
    {
        if b then
            return with a
    }

    var i : integer

    i := test(2, true)
}