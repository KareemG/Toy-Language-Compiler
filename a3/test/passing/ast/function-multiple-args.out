 { 
 function test(a : integer, b : boolean) : integer { 
 if b then
 return with aend
 } 
 var i : integer
 i : = test(2, true)
 }
